docs/readme.md

```diff
@@ -0,0 +1,109 @@
+Useful documentation, examples, and [recipes](./recipes/) to get you started.
```

Comment:
well... im okay with it being `readme`. but let's add one more link to `architecture.md` at the top or so.